report-viewer/src/vue-virtual-scroll.d.ts

```diff
@@ -0,0 +1,21 @@
+// Type definitions for vue-virtual-scroller
```

Comment:
I understand that this is required, but please add a comment linking to the [original issue](https://github.com/Akryum/vue-virtual-scroller/issues/199).